js/main.js

```diff
@@ -196,9 +198,19 @@ function pokemonFlavorText(id) {
     pPokemonFT.appendChild(tnPokemonFT);
     divPokemonPage.appendChild(pPokemonFT);
 
-    var buttonNotCaught = document.createElement('button');
-    buttonNotCaught.setAttribute('class', 'not-caught');
-    divPokemonPage.appendChild(buttonNotCaught);
+    var buttonCaught = document.createElement('button');
+    for (var z = 0; z < data.caughtList.length; z++) {
+      if (data.caughtList[z].pokemon_name === id) {
+        buttonCaught.setAttribute('class', 'catch caught');
+        break;
+      } else {
+        buttonCaught.setAttribute('class', 'catch not-caught');
+      }
+    }
+    divPokemonPage.appendChild(buttonCaught);
+
+    $caughtButton = document.querySelector('.catch');
```

Comment:
Is this necessary? Doesn't `buttonCaught` already hold this element?

Reply:
If it's just a matter of updating the value of `$caughtButton`, I would just do that on line 201 instead of making a new variable to hold the element 🤓